### DISCUSSION TITLE
Is there a way to input the .m3u and get the guide (or at least the file to make the guide)?

### DISCUSSION THREAD

**Discussion by cinderblockgames** [iptv-org/community#1043] [Q&A]
If I'm grabbing hundreds of channels off https://github.com/iptv-org/iptv, what's the easiest way to get the guide info for them?  I'd have expected to be able to point to something (e.g., the .m3u itself) to put together what I need for the guide, but the readme here makes it seem basically manual to put together.  Am I missing the easy button?

**Comment by freearhey** [comment]
Unfortunately we cannot provide such a “button” at this time.
